packages/x-date-pickers-pro/src/DateRangePickerDay/DateRangePickerDay.tsx

```diff
@@ -50,6 +50,11 @@ export interface DateRangePickerDayProps<TDate extends PickerValidDate>
    * Indicates if the day should be visually selected.
    */
   isVisuallySelected?: boolean;
+  /**
+   * If `true`, the day can be dragged to change the current date range.
+   * @default false
+   */
+  draggable?: boolean;
```

Comment:
Right now the typing comes from the DOM element which also accepts `"true"` and `"false"`
But we are only passing `true` of `undefined` so I restricted the typing so that our check in the styled component and the typing are coherent